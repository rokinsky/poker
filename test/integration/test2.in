texas-holdem KdKsJcJhJs AdAc KhTc
texas-holdem 7d8sJcTs9h KdQc 7h7c
texas-holdem 2d5s5c9hTs KdQs KhJc
texas-holdem 6h3d5c7s4s 6s7c 4d5d
texas-holdem 2h5c8sAsKc KdQh 3cKh
texas-holdem 5d5h6hQdTd 7s3c Ts3s 9c7d 8h3d 9h9s AsKd 4dQs TcJd
texas-holdem 4cKs4h8s7s Ad4s Ac4d As9s KhKd 5d6d
texas-holdem 2h3h4h5d8d KdKs 9hJh
texas-holdem JcTc9c8cAd Qc4h 7c8h
texas-holdem AdAc8d8c7c AhAs 8s8h
texas-holdem AdAcAhAs7c 9s8d JsTd
texas-holdem Ah4h2d7sTc TsTh AdAc
texas-holdem Ah4hAsAc5d QcJh Qh7s
texas-holdem QcQhTs8s3h Td9c 3cJs
texas-holdem QcQhTs8s3h Td9c TcJs
texas-holdem AhKs8s6d5c 2cQh Js4d
texas-holdem AsTd8d6c4h KsQh 7dJc
texas-holdem KdQd8c6c5c AcAh JdTd 6h6d 2cJc
texas-holdem KdQd8c6c5c AcAh JdTd 6h6d 2cJc
texas-holdem 5cAcAdAh5h 7c7d 9s5s
texas-holdem 5cAcAdAh5h 7c7d 9s4s
texas-holdem 6c7c8c9cKd TcQs 5c4d
texas-holdem 5h6h7h8h9h TcQs 5c4d
texas-holdem QdQsQhQc5d KhKd Ac2c
texas-holdem QdQsQhQc5d KhAd Ac2c
texas-holdem AsKsTs7s5s 2d3s 3d6s
texas-holdem 8s9cJd7h2s ThJs TcQh
texas-holdem Ks9cTh2d6c TsKd Kh9s
texas-holdem AdKd5s9c8c KhTc Ah3s QhQd
texas-holdem 5d5c6c6hKd 4c4h Ac9d
texas-holdem 5d5c6c6hKd 4c4h 8c9d
texas-holdem TsJsQsKsAs Ad5d 6dKd
texas-holdem 8s9sTs7s2d 6s6d JsJd
texas-holdem 8s9dTs7h2d 6s6d JsJd
texas-holdem 2s3d4s5h6d 6s6h JsJd
texas-holdem 2s3d4s5h8d As8h 6sJd
texas-holdem 2s3s4s5s8d As8h 6sJd
texas-holdem 8d8h4sQsQd Qh7h 8sJd
texas-holdem 8d8h9sQsQd Qh7h Qc9d
texas-holdem 8d8h8sQsQd 9c7h Jc9d
texas-holdem 4cKs4h8s7s Ad4s Ac4d As9s KhKd 5d6d
texas-holdem 2h3h4h5d8d KdKs 9hJh
texas-holdem AsAcAhAdKd KcKs QsQc
texas-holdem AdAc8d8c7c AhAs 8s8h
texas-holdem AsAcAhAdKh KdQs KcKs
texas-holdem 7s4d6c3h5h 7cAc Js5s
texas-holdem TsJdQcThAh 7c8c 8s8h
texas-holdem 5s7h5hAh7s As5d 7d8s Jh9h
texas-holdem JsJd8c8h8s 7s7d TcKs
texas-holdem 3s7d3c5h7h Ah5d Ad9s
texas-holdem 3sTd3hKc2h Ah5d Ad9s
texas-holdem 3s7d3hJc2h AhKd AdQs
texas-holdem ThJhKhJc2h AhAs AdAc
texas-holdem ThJdKh3c2h 3h3s AdQs
texas-holdem TsJs3d3cKs 3h3s AsQs
texas-holdem 2sJs3s3c2h 8s9s 7sTs
texas-holdem AdKs5cJhQd KdQs KhJc
texas-holdem 8d8s9cTh9s KcQc 7h7c
texas-holdem 6d8sJc9hTs QdAc 7h7c
texas-holdem 6d8sJc9hTs AdAc 7h7c
texas-holdem Kc9sThKdTs As3s Ad9d
texas-holdem ThJdKhJc2h Ah3h AdQs
texas-holdem 2sKs3sAsJs 8s9s 6sTs
texas-holdem 9sKs8sAsJs 2s6s 3s7s
texas-holdem 8dKs5c3h9s KdQs KhJc
