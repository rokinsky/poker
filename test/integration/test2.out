AdAc KhTc
7h7c KdQc
KhJc KdQs
4d5d=6s7c
3cKh KdQh
7s3c 8h3d 9c7d AsKd 9h9s TcJd=Ts3s 4dQs
Ac4d=Ad4s 5d6d As9s KhKd
KdKs 9hJh
7c8h Qc4h
8s8h AhAs
9s8d JsTd
TsTh AdAc
Qh7s QcJh
3cJs Td9c
Td9c TcJs
Js4d 2cQh
7dJc KsQh
JdTd AcAh 6h6d 2cJc
JdTd AcAh 6h6d 2cJc
9s5s 7c7d
9s4s 7c7d
5c4d TcQs
5c4d=TcQs
KhKd Ac2c
Ac2c=KhAd
2d3s 3d6s
ThJs TcQh
Kh9s TsKd
QhQd KhTc Ah3s
4c4h Ac9d
4c4h=8c9d
6dKd=Ad5d
6s6d JsJd
6s6d JsJd
6s6h=JsJd
As8h 6sJd
As8h 6sJd
8sJd Qh7h
Qh7h Qc9d
9c7h=Jc9d
Ac4d=Ad4s 5d6d As9s KhKd
KdKs 9hJh
KcKs=QsQc
8s8h AhAs
KcKs=KdQs
7cAc=Js5s
7c8c 8s8h
Jh9h As5d 7d8s
7s7d=TcKs
Ad9s Ah5d
Ad9s=Ah5d
AdQs AhKd
AdAc AhAs
3h3s AdQs
3h3s AsQs
8s9s 7sTs
KhJc KdQs
7h7c KcQc
7h7c QdAc
AdAc 7h7c
Ad9d=As3s
AdQs Ah3h
8s9s 6sTs
2s6s=3s7s
KhJc KdQs
